src/lint-gazetteer.js

```diff
@@ -0,0 +1,101 @@
+const geojsonhint = require('@mapbox/geojsonhint');
+
+function lint(gazetteer) {
```

Comment:
@aparlato I don't imagine that this is how the lint function will be used— rather, it would be used on an arbitrary geojson file _not_ provided by the gazetteer project, eg. if you're creating a UI in which you can upload custom gazetteer